### ISSUE TITLE
Set number of timesteps to 12 in `test_optical_flow_data_source.py`

### ISSUE THREAD

**Issue by JackKelly** [openclimatefix/nowcasting_dataset#513]
## Detailed Description
As a nasty hack, I've set the number of timesteps to zero because of issue #511.

After #512 and/or #511 is closed, we should set the number of timesteps back to 12.

**Comment by JackKelly** [comment]
no longer relevant due to new design of OpticalFlowDataSource
